src/megaclient.cpp

```diff
@@ -9110,8 +9117,8 @@ void MegaClient::notifynode(Node* n)
         {
             // report a "NO_KEY" event
 
-            char* buf = new char[n->nodekey.size() * 4 / 3 + 4];
-            Base64::btoa((byte *)n->nodekey.data(), int(n->nodekey.size()), buf);
+            char* buf = new char[n->nodekey().size() * 4 / 3 + 4];
+            Base64::btoa((byte *)n->nodekey().data(), int(n->nodekey().size()), buf);
```

Comment:
We should update this at some point, but for this PR I think it's best to keep the change simple by just using the accessor instead of the member directly here, that way there's no possible mistake